middleware/session/README.md

```diff
@@ -76,7 +76,8 @@ app.Get("/", func(c *fiber.Ctx) error {
 		panic(err)
 	}
 
-	return fmt.Fprintf(ctx, "Welcome %v", name)
+	_, err = fmt.Fprintf(c, "Welcome %v", name)
```

Comment:
Maybe we should use c.SendString? Just to make it a bit simpler and use Fiber functions

Reply:
Jo, i will change it tomorrow